pom.xml

```diff
@@ -52,6 +52,7 @@
     <spring.version>4.1.4.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-transform.version>4.4.0</fcrepo-transform.version>
+    <fcrepo-java-client.version>4.4.1-SNAPSHOT</fcrepo-java-client.version>
```

Comment:
@mohideen see: https://jira.duraspace.org/browse/FCREPO-1858 and the two corresponding PRs which address this.